google/oauth2/credentials.py

```diff
@@ -351,33 +350,6 @@ def with_universe_domain(self, universe_domain):
     def _metric_header_for_usage(self):
         return metrics.CRED_TYPE_USER
 
-    def _set_account_from_access_token(self, request):
```

Comment:
Could you alternatively perform this in the `account` property in case `_account` is not already set?